programs/streamflow-sdk/src/lib.rs

```diff
@@ -124,14 +126,98 @@ declare_id!("HqDGZjaVRXJ9MGRQEw7qDc2rAr6iH1n1kAQdCZaCMfMZ");
+///     #[account(
+///         init_if_needed,
+///         payer = sender,
+///         associated_token::mint = mint,
+///         associated_token::authority = streamflow_treasury,
+///     )]
+///     /// CHECK: Associated token account address of `streamflow_treasury`.
+///     pub streamflow_treasury_tokens: Box<Account<'info, TokenAccount>>,
+///     #[account(mut)]
+///     /// CHECK: Delegate account for automatically withdrawing contracts.
+///     pub withdrawor: UncheckedAccount<'info>,
+///     #[account(mut)]
+///     /// CHECK: Partner treasury account.
+///     pub partner: UncheckedAccount<'info>,
+///     #[account(
+///         init_if_needed,
+///         payer = sender,
+///         associated_token::mint = mint,
+///         associated_token::authority = partner,
+///     )]
+///     pub partner_tokens: Box<Account<'info, TokenAccount>>,
+///     pub mint: Box<Account<'info, Mint>>,
+///     /// CHECK: Internal program that handles fees for specified partners.
+///     pub fee_oracle: UncheckedAccount<'info>,
+///     pub rent: Sysvar<'info, Rent>,
+///     /// CHECK: Streamflow protocol (alias timelock) program account.
+///     pub streamflow_program: UncheckedAccount<'info>,
+///     pub token_program: Program<'info, Token>,
+///     pub associated_token_program: Program<'info, AssociatedToken>,
+///     pub system_program: Program<'info, System>,
+/// }
 /// ```
 
 
 #[program]
 pub mod streamflow_sdk {
     use super::*;
 
-    /// Anchor rpc handler used for CPI code generation
+    /// Create a Stream
+    ///
+    /// # Arguments
+    ///
+    /// * `ctx` - Accounts that will be used on Stream creation
+    /// * `start_time` - Unix Timestamp for Stream start, can be 0 to use current time
+    /// * `net_amount_deposited` - Amount of Tokens to deposit to the Stream
+    /// * `period` - Unlock Period in Seconds, tokens will be unlocked every `period` seconds
+    /// * `amount_per_period` - Unlock Amount, every `period` we unlock `amount_per_period` tokens
+    /// * `cliff` - Unix Timestamp of Cliff (first unlock), can be 0 to use current time or not use at all
+    /// * `cliff_amount` - Cliff Amount of tokens, can 0 to not use Cliff at all
+    /// * `cancelable_by_sender` - Whether Stream can by cancelled by Sender
+    /// * `cancelable_by_recipient` - Whether Stream can be cancelled by Recipient
+    /// * `automatic_withdrawal` - Whether automatic withdrawals are enabled
+    /// * `transferable_by_sender` - Whether Stream can be transferred by Sender
+    /// * `transferable_by_recipient` - Whether Stream can be transferred by Recipient
+    /// * `can_topup` - Whether Stream can be topped up (deposit additional tokens)
+    /// * `stream_name` - Name of the Stream
+    /// * `withdraw_frequency` - Frequency of Automatic Withdrawals, if `automatic_withdrawal` is on, every `withdraw_frequency` seconds **all unlocked** tokens will be sent to the recipient
```

Comment:
I'd say `Frequency of Automatic Withdrawals,` is unnecessary, because it's obvious from the title.